src/Util/Process.php

```diff
@@ -36,9 +39,31 @@ public static function run(
         string|null $workingDirectory = null,
         int|null $timeout = 5,
         callable|null $outputCallback = null,
+        bool $cleanWarningsAndDeprecations = false,
     ): string {
-        return trim((new SymfonyProcess($command, $workingDirectory, timeout: $timeout))
+        $output = trim((new SymfonyProcess($command, $workingDirectory, timeout: $timeout))
             ->mustRun($outputCallback)
             ->getOutput());
+
+        if ($cleanWarningsAndDeprecations) {
+            return self::cleanLanguageWarningAndDeprecations($output);
+        }
+
+        return $output;
+    }
+
+    private static function cleanLanguageWarningAndDeprecations(string $testOutput): string
+    {
+        $testOutput = explode("\n", $testOutput);
```

Comment:
This might need to be `PHP_EOL`; I don't think we have any tests to reproduce this at the moment, so I don't actually know if Windows would use `\r\n` in the process output. Worth verifying with a test IMO!

Reply:
Added a behavior test that tries to load an invalid extension. Works well 👍